Sources/Plasma/FeatureLib/pfConsole/pfConsoleCommands.cpp

```diff
@@ -1357,6 +1357,10 @@ PF_CONSOLE_CMD( Graphics_Renderer, Gamma2, "float g", "Set gamma value (alternat
             float sinT = std::sin(t * hsConstants::pi<float> / 2.f);
 
             float remap = std::clamp(t + (sinT - t) * g, 0.f, 1.f);
+            if( remap < 0 )
+                remap = 0;
+            else if( remap > 1.f )
+                remap = 1.f;
```

Comment:
Was the existing `std::clamp` call not actually clamping this properly?

Reply:
Iiiiiiii don't remember. This is kind of odd. I'll check out why this change was made.

Reply:
This may have been a merge issue. I think this code originally didn't use std::clamp. Then when it went through its own PR it switched to std::clamp. Then that PR got merged back with this branch and then it did both. Removing this chunk.